ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__azure-4.17-nightly-x86.yaml

```diff
@@ -126,6 +137,26 @@ tests:
     - chain: openshift-qe-control-plane
     workflow: openshift-qe-installer-azure
   timeout: 5h0m0s
+- always_run: false
+  as: control-plane-multi-ipsec-3nodes
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: azure-arm64
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
```

Comment:
We need ADDITIONAL_WORKERS to specify the arm type instance account to 1, because the default is 2 in ref ipi-install-heterogeneous